code/game/objects/items/weapons/grenades/plastic.dm

```diff
@@ -78,6 +102,12 @@
 		src.target = AM
 		forceMove(null)	//Yep
 
+		if(istype(AM, /obj/item)) //your crappy throwing star can't fly so good with a giant brick of c4 on it.
+			var/obj/item/I = AM
```

Comment:
>They said none will use it

they (they being remie who made embedding) also said it was intentional emergent gameplay